packages/clerk-js/src/ui/components/OrganizationProfile/VerifiedDomainPage.tsx

```diff
@@ -175,6 +220,16 @@ export const VerifiedDomainPage = withCardStateProvider(() => {
               direction={'col'}
               gap={4}
             >
+              {calloutLabel.length > 0 && (
+                <CalloutWithAction icon={InformationCircle}>
+                  {calloutLabel.map((label, index) => (
+                    <React.Fragment key={index}>
+                      {label}
+                      <br />
+                    </React.Fragment>
+                  ))}
+                </CalloutWithAction>
+              )}
```

Comment:
Is there a need for this to be so dynamic? Can we simply list the 3 texts manually here and remove useCalloutLabel altogether?